src/hypergraph.jl

```diff
@@ -114,6 +151,14 @@ If a vertex does not belong to a hyperedge `nothing` is returned.
     get(h.v2he[idx[1]], idx[2], nothing)
 end
 
+@inline function Base.pop!(d::SortedDict,key::Int,::Nothing)
```

Comment:
this should be a PR do DataStructures.jl as doing it this way is type piracy

Reply:
done: https://github.com/JuliaCollections/DataStructures.jl/pull/566 
Let's see if this gets accepted than we can drop that function from here.